core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/command/AbstractCommandTriggerManager.java

```diff
@@ -35,14 +35,6 @@
 
 
 public abstract class AbstractCommandTriggerManager extends AbstractTriggerManager<CommandTrigger> {
-    private static final String SYNC = "sync";
-    private static final String PERMISSION = "permissions";
-    private static final String ALIASES = "aliases";
-    public static final String TABS = "tabs";
-
-    public static final String HINT = "hint";
-    public static final String CANDIDATES = "candidates";
```

Comment:
We should offer a solution to fix inconsistent keys without users having to do do anything else.